includes/class-wc-geolocation.php

```diff
@@ -216,11 +216,6 @@ public static function geolocate_ip( $ip_address = '', $fallback = true, $api_fa
 				} else {
 					$country_code = '';
 				}
-
-				if ( ! $country_code && $fallback ) {
```

Comment:
We should deprecate the `$fallback` variable in the function args, docblock, etc. since it's not used for anything after this change.

An alternate way of handling this change without needing a deprecated function arg would be to make `$fallback = false` the default and [also update it to `false` here](https://github.com/woocommerce/woocommerce/blob/00b48febace054f293dc2f73d0f7ebcb993d26ac/includes/wc-core-functions.php#L1082).